cdk/lib/constructs/embedding.ts

```diff
@@ -42,18 +42,22 @@ export class Embedding extends Construct {
   constructor(scope: Construct, id: string, props: EmbeddingProps) {
     super(scope, id);
 
+    const embedding_container_vcpu:number = Stack.of(this).node.tryGetContext("embedding_container_vcpu") ? Stack.of(this).node.tryGetContext("embedding_container_vcpu") : 1024
```

Comment:
Please read the context value in `bin/bedrock-chat.ts` for maintainability (and also please declare parameters as well)

Reply:
fixed